powermon/formats/simple.py

```diff
@@ -20,29 +21,29 @@ def format(self, result: Result) -> list:
         _result = []
 
         # check for error in result
-        if result.error:
-            data = {}
-            data["Error"] = [f"Command: {result.command_code} incurred an error or errors during execution or processing", ""]
-            data["Error Count"] = [len(result.error_messages), ""]
-            for i, message in enumerate(result.error_messages):
-                data[f"Error #{i}"] = [message, ""]
-        else:
-            data = result.decoded_responses
-
-        if data is None:
+        #if result.error:
```

Comment:
wont this mean errors dont get displayed?

Reply:
The errors are being added to the responses through exceptions at other places in the code.